src/api.py

```diff
@@ -49,28 +76,15 @@ def blob():
                 try:
                     new_blob_info = BlobPostSchema.parse_obj(request.get_json()) 
                     blob_type = new_blob_info.metadata['blob_type']
-                    if not blob_type in BLOB_TYPES.keys():
```

Comment:
bring this validation back.

Reply:
https://docs.pydantic.dev/usage/validators/

Reply:
used pydantic  @validator to check this